src/Search/Search.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { findByTestAttr } from '../../__tests__/Utils';
+import { Search } from './Search';
+
+const renderSearch = (props = {}) => {
+  const component = shallow(<Search {...props} />);
+  return component;
+};
+
+describe('should render Search', () => {
```

Comment:
I don't think it's good idea to have several root level `describe` blocks

Reply:
Updated